crypto/compactcert/const.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package compactcert
+
+import (
+	"github.com/algorand/go-algorand/crypto"
+)
+
+// HashType/ hashSize relate to the type of hash this package uses.
+const (
+	HashType        = crypto.Sumhash
+	HashSize        = crypto.SumhashDigestSize
+	SignatureScheme = crypto.FalconType
+)
+
+const (
+	// MaxReveals is a bound on allocation and on numReveals to limit log computation
+	MaxReveals = 1024
+
+	// MaxProofDigests is a bound on allocation on number of proofs
+	MaxProofDigests = 20 * MaxReveals
```

Comment:
What is the purpose of `MaxProofDigests`? Where does "20" come from?
I think `MaxProofDigests` is not used.

Reply:
I think you are right and we might not need it (because we have a bound on the proof). we will absolutely review this once stateproof are released. for now, this is unused